### DISCUSSION TITLE
Anyway to invert the direction of the y-axis?

### DISCUSSION THREAD

**Discussion by sham3n** [patrykandpatrick/vico#909] [Questions]
### Question

Anyway to invert the y-axis so that the highest value will be on the bottom and the lowest value will be on top?

### Vico version(s)

V2.0.0-beta.1

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @sham3n. There isn't a dedicated property for this currently, but you can implement it easily by using negative _y_ values and inverting them during formatting.
